PlayCover/Views/AppSettingsView.swift

```diff
@@ -141,6 +141,9 @@ struct GraphicsView: View {
                         Text("iPad Pro (12.9-inch) (1st gen) | A9X | 4GB").tag("iPad6,7")
                         Text("iPad Pro (12.9-inch) (3rd gen) | A12Z | 4GB").tag("iPad8,6")
                         Text("iPad Pro (12.9-inch) (5th gen) | M1 | 8GB").tag("iPad13,8")
+                        Divider()
+                        Text("iPhone 14 Pro Max | A16 | 6GB").tag("iPhone15,3")
```

Comment:
A purely aesthetic observation, currently the iPad models are shown in an increasing order. Perhaps the iPhone models should be too, or have both in a descending order.

Reply:
sure